test/integration/generated/nonStringEnum/src/models/index.ts

```diff
@@ -11,11 +11,11 @@ import * as coreHttp from "@azure/core-http";
 /**
  * Defines values for IntEnum.
  */
-export type IntEnum = 200 | 403 | 405 | 406 | 429 | string;
+export type IntEnum = 200 | 403 | 405 | 406 | 429 | number;
```

Comment:
Are we updating the mappers for these as well?

Reply:
In the mapper the type will be set to number

Reply:
I would expect this not to be a string:

https://github.com/Azure/autorest.typescript/blob/c57733992fb0ba8917dd03b735dc004459b6e1aa/test/integration/generated/nonStringEnum/src/models/parameters.ts#L70-L75

Reply:
Agreed. Give me few minutes. I will do it

Reply:
Fixed it